extensions/interactions/MultipleChoiceInput/protractor.js

```diff
@@ -36,14 +36,9 @@ var customizeInteraction = function(elem, richTextInstructionsArray) {
 // the options.
 var expectInteractionDetailsToMatch = function(
     elem, richTextInstructionsArray) {
-  elem.all(by.repeater('choice in $ctrl.choices track by $index'))
+  elem.all(by.repeater('choice in $ctrl.choices'))
     .then(function(optionElements) {
       expect(optionElements.length).toEqual(richTextInstructionsArray.length);
-      for (var i = 0; i < optionElements.length; i++) {
```

Comment:
You cannot just drop this check -- it weakens the tests. You'll need some kind of check to extract the values, sort them, and compare that to the expected sorted array.

Reply:
I have changed it accordingly. Done.